simplecalculator/calculator.py

```diff
@@ -61,7 +61,7 @@ async def calculate(self, ctx, *, query):
         e = discord.Embed(
             color=await ctx.embed_color(),
             title=f"Input: `{query}`",
-            description=f"Output: `{output_string}`",
+            description=f"Output: `{float(output_string):,.2f}`",
```

Comment:
The output from TSE isn't guaranteed to be numeric, so you need to modify this so that it catches the ValueError if it fails cast to float. Additionally, the output shouldn't lose precision when formatted.

Reply:
Can i ask in which case wont the Output be numeric?

Reply:
![something](https://user-images.githubusercontent.com/57430307/148767151-30d44e55-0fe0-40af-b252-8ef715d1bc88.png)

Reply:
ahh ok thanks. Ill get around to handling that soon